docs/Introduction.md

```diff
@@ -7,27 +7,28 @@ To illustrate the problems React Router is going to solve for you, let's build a
 ### Without React Router
 
 ```js
-import React from 'react'
+import React, { Component } from 'react'
+import ReactDOM from 'react-dom'
 
 const About = React.createClass({/*...*/})
 const Inbox = React.createClass({/*...*/})
 const Home = React.createClass({/*...*/})
```

Comment:
The actual React Router code uses `React.createClass` everywhere, though, no? It might be better to be consistent with that.